core/src/action.rs

```diff
@@ -83,7 +89,7 @@ pub enum Action {
 
     /// ???
     // @TODO how does this relate to validating a commit?
-    ValidateEntry(EntrySubmission),
+    ValidateAppEntry(EntrySubmission),
```

Comment:
Is there something else than an AppEntry?

Reply:
I'd be happy to know if you have a better name for non-system entry types.
I didn't have a clear preference between: Data, App, User, Custom, Zome

Reply:
Why not call them `Entry`?
And then we also have SystemEntries..

Reply:
I'll try explaining with pseudo Math:
`Entry = SystemEntry u AppEntry`
So if you call AppEntry 'Entry' you get:
`Entry = SystemEntry u Entry`
Which is confusing.

I'm going under the assumption that some operations/actions are done on AppEntries only, like Validation.
And some operations are done on any Entry, like Put in HashTable.
So I would like the name of the operations to reflect that.

Reply:
Hm, SystemEntries have to be validated as well, no? @zippy
Just differently than AppEntries.

I'd say in pseudo Math:
`SystemEntry c Entry`